pkgs/racket-doc/scribblings/reference/compiler.scrbl

```diff
@@ -136,4 +139,6 @@ compilation and evaluation times on exit. When the
 information about macro-expansion time is printed on exit.
 
 @history[#:changed "8.8.0.10" @elem{Added special pass name @tt{all}
-                                    to @envvar{PLT_LINKLET_SHOW_PASSES}}]
+                                    to @envvar{PLT_LINKLET_SHOW_PASSES}}
+         #:changed "8.11.1.2" @elem{Added module and linklet info
+                                    to output}]
```

Comment:
A nit, but if you're updating docs anyway for a typo, each of theses `elems` (the old one and new one) should end with a period.